.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -30,20 +39,32 @@ body:
     validations:
       required: true
 
-  - type: textarea
+  - type: dropdown
```

Comment:
I'm not convinced that a dropdown is the best choice here since the version of the OS/browser is lost. I'd suggest either listing every supported major versions of each OS/browser, or providing multiple text fields (one for the OS, one for the browser, etc.) so that the bug reporter *can* provide a more thorough description of their env.